pkg/kev/config/schema.go

```diff
@@ -0,0 +1,60 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+var ServicesSchema = map[string]interface{}{
+	"$schema": "http://json-schema.org/draft/2019-09/schema#",
+	"$id":     "http://appvia.io/schemas/kev-service-labels-schema.json",
+	"type":    "object",
+	"properties": map[string]interface{}{
+		LabelComponentEnabled:                  map[string]interface{}{"type": "string", "enum": []string{"false", "true"}},
+		LabelServiceType:                       map[string]interface{}{"type": "string", "enum": []string{"None", "Headless", "ClusterIP", "Nodeport", "LoadBalancer"}},
+		LabelWorkloadCPU:                       map[string]interface{}{"type": "string"},
+		LabelWorkloadImagePullPolicy:           map[string]interface{}{"type": "string", "enum": []string{"IfNotPresent", "Always"}},
+		LabelWorkloadLivenessProbeCommand:      map[string]interface{}{"type": "string"},
+		LabelWorkloadLivenessProbeDisabled:     map[string]interface{}{"type": "string", "enum": []string{"false", "true"}},
+		LabelWorkloadLivenessProbeInitialDelay: map[string]interface{}{"type": "string", "format": "duration"},
+		LabelWorkloadLivenessProbeInterval:     map[string]interface{}{"type": "string", "format": "duration"},
+		LabelWorkloadLivenessProbeRetries:      map[string]interface{}{"type": "string", "pattern": `^\d+$`},
+		LabelWorkloadLivenessProbeTimeout:      map[string]interface{}{"type": "string", "format": "duration"},
+		LabelWorkloadMaxCPU:                    map[string]interface{}{"type": "string"},
+		LabelWorkloadMaxMemory:                 map[string]interface{}{"type": "string", "pattern": `^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$`},
+		LabelWorkloadMemory:                    map[string]interface{}{"type": "string", "pattern": "^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$"},
+		LabelWorkloadReplicas:                  map[string]interface{}{"type": "string", "pattern": `^\d+$`},
+		LabelWorkloadRollingUpdateMaxSurge:     map[string]interface{}{"type": "string", "pattern": `^\d+$`},
+		LabelWorkloadServiceAccountName:        map[string]interface{}{"type": "string", "pattern": `^[a-zA-Z0-9._-]+$`},
+		LabelWorkloadType:                      map[string]interface{}{"type": "string", "enum": []string{"Pod", "Deployment", "StatefulSet", "Daemonset", "Job"}},
+	},
+	"required": []string{
+		LabelWorkloadLivenessProbeCommand,
+		LabelWorkloadReplicas,
+	},
+	"additionalProperties": false,
+}
+
+var VolumesSchema = map[string]interface{}{
+	"$schema": "http://json-schema.org/draft/2019-09/schema#",
+	"$id":     "http://appvia.io/schemas/kev-service-labels-schema.json",
+	"type":    "object",
+	"properties": map[string]interface{}{
+		LabelVolumeSize:         map[string]interface{}{"type": "string", "pattern": `^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$`},
```

Comment:
Comment as above.

Reply:
Although it makes sense to allow T or even P in this case.

Reply:
Same as above.

Reply:
Left the regex as is.